src/wui/wui_helper_funcs.c

```diff
@@ -1,26 +1,25 @@
 #include "wui_helper_funcs.h"
-#include "dbg.h"
-#include "wui.h"
 #include "string.h"
+#include "dbg.h"
 
 extern osMessageQId tcp_wui_mpool_id;
 extern osSemaphoreId tcp_wui_semaphore_id;
 
-void send_request_to_wui(const char *request) {
+uint8_t send_request_to_wui(wui_cmd_t * req_ptr) {
```

Comment:
why uint8_t ? why not uint32_t?

Reply:
ret val will be used to trigger "not enough spaci in osPool", so it should probably be HTTPC_COMMAND_STATUS instead

Reply:
i meant data type